src/modules/AmidoBuild/exported/Build-DockerImage.Tests.ps1

```diff
@@ -139,4 +139,31 @@ Describe "Build-DockerImage" {
             $Session.commands.list[2] | Should -BeLike "*docker* push docker.io/pester-tests:unittests"
         }
     }
+
+    Context "Build image and push including latest" {
+
+        BeforeEach {
+            # Reset the commands list to an empty array
+            $global:Session.commands.list = @()            
+        }
+        
+        It "will build and push the image to the specified registry with a latest tag" {
+
+            # Call the function under test
+            Build-DockerImage -name pester-tests -tag "unittests" -registry "docker.io" -push -latest
+
+            # Check the build command
+            $Session.commands.list[0] | Should -BeLike "*docker* build . -t pester-tests:unittests -t docker.io/pester-tests:unittests -t docker.io/pester-tests:latest"
+
+            # Check that docker logs into the registry
+            $Session.commands.list[1] | Should -BeLike "*docker* login docker.io -u pester -p pester123"
+
+            # Ensure that the image is pushed to the registry with specific tag
+            $Session.commands.list[2] | Should -BeLike "*docker* push docker.io/pester-tests:unittests"
+
+            # Ensure that the image is pushed to the registry with latest tag
+            $Session.commands.list[3] | Should -BeLike "*docker* push docker.io/pester-tests:latest"
+        }
+    }
+
 }
```

Comment:
Newline missing, you might want to configure your IDE to add them

Reply:
Note: Maybe some form of lining around it, but not sure the best way currently.

Reply:
Done and done, cheers!